app/ui/legacy/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -1707,6 +1731,20 @@ private void setContextCapabilities(Account account, Menu menu) {
                     menu.findItem(R.id.spam).setVisible(false);
                 }
             }
+
+            // if currentFolder is outbox: reduce available menu items to 'move to draft' and 'delete'
+            if(isOutbox()) {
+                menu.findItem(R.id.mark_as_read).setVisible(false);
+                menu.findItem(R.id.mark_as_unread).setVisible(false);
+                menu.findItem(R.id.archive).setVisible(false);
+                menu.findItem(R.id.copy).setVisible(false);
+                menu.findItem(R.id.flag).setVisible(false);
+                menu.findItem(R.id.unflag).setVisible(false);
+                menu.findItem(R.id.spam).setVisible(false);
+
+                MenuItem moveMenuItemView = (MenuItem) menu.findItem(R.id.move);
+                moveMenuItemView.setTitle(R.string.move_to_draft_action);
```

Comment:
We try to avoid changing menu items in code. Instead, please add a new menu item `move_to_drafts` that is only displayed for the Outbox folder (and only when a Drafts folder is configured, see `Account.hasDraftsFolder()`.)